lib/p2p/errors.ts

```diff
@@ -42,6 +43,10 @@ const errors = {
     message: `could not connect to peer at ${addressUtils.toString(address)}: ${err.message}`,
     code: errorCodes.COULD_NOT_CONNECT,
   }),
+  NODE_NOT_FOUND: (nodePubKey: string) => ({
+    message: `could not find node ${nodePubKey}`,
```

Comment:
For the wording here, what do you think of `NODE_UNKNOWN`? I think that's a bit more descriptive, although "not found" is also correct.